keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -1507,9 +1515,18 @@ spec:
 ### Evaluation
 
 We use the `syncJob` function of the Job controller to evaluate the specified
-`podFailurePolicy` rules against the failed pods. It is only the first rule with
-matching requirements which is applied as the rules are evaluated in order. If
-the pod failure does not match any of the specified rules, then default
+`podFailurePolicy` rules against the failed pods.
+
+Since terminating Pods (have `deletionTimestamp` and are not `Failed` or
+`Succeeded`) don't have an exit code yet and might actually succeed, the 
```

Comment:
nit: Not sure it changes much, but aside from exit codes terminating pods may yet have the DisruptionTarget condition added.

Reply:
we generally add the condition before issuing the deletion, so in general it's just about the exit code. I'll leave it like this for now.

Reply:
Sgtm